README.md

```diff
@@ -1,2 +1,156 @@
-# hello_Microverse
-My first repository on Microverse
+<a name="readme-top"></a>
+
+
+
+<div align="center">
+  <h1><b>README File</b></h3>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 Hello Microverse <a name="about-project"></a>
+
+
+
+**Hello Microverse** is my first proyect as a Microverse Student using Git & Github.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> - HTML, Git, CSS
+
+<details>
+  <summary>HTML</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/es/docs/Web/HTML">Definition</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>CSS</summary>
+  <ul>
+    <li><a href="https://es.wikipedia.org/wiki/CSS">Definition</a></li>
+  </ul>
+</details>
+
+
+
+### Key Features <a name="key-features"></a>
+
+- **Add linters to repository**
+- **Use of GitFlow**
+- **Add README file**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- Visual Studio Code or IDE
+- Git
+- Github account
+- NodeJS
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+### Install
+
+Installation not required
+
+
+### Usage
+
+To run the project, execute it with an IDE or VS code
+
+
+### Run tests
+
+Running tests not required
+
+
+### Deployment
+
+To deploy use your favorite method
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+> by Rodolfo Carrillo
+
+👤 **Rodolfo Carrillo**
+
+- GitHub: [@rudicarrilloypr](https://github.com/rudicarrilloypr)
+- Twitter: [@onlyargo](https://twitter.com/onlyargo)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/rudi-carrillo-957964249/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Coming Soon**
```

Comment:
- [ ] As stated by the previous reviewer, Please update this section to include 1-3 features you plan to add to this project in the future. This section is very important as it informs potential future collaborators on what they can collaborate with you on💪